torch/_dynamo/guards.py

```diff
@@ -100,6 +100,29 @@ def strip_getattr_getitem(name):
     return re.split(r"[.\[]", name)[0]
 
 
+# Don't put anything on here you don't want laundered to the next frame
```

Comment:
It would be better to just explain what this data structure actually does

Reply:
Agreed.